ref-farming/src/simple_farm.rs

```diff
@@ -292,17 +319,41 @@ impl SimpleFarm {
         (self.last_distribution.rps, claimed)
     }
 
-    pub fn can_be_removed(&self) -> bool {
+    /// Move an Ended farm to Cleared, if any unclaimed reward exists, go to beneficiary
+    pub(crate) fn move_to_clear(&mut self, total_seeds: &Balance) -> bool {
```

Comment:
who can call move_to_clear?

Reply:
Only the owner of this contract, that is the DAO can call it.   

The backend thoughts is we need a way to clean those ended farms especially those lasts for a long time, to save some gas fee when perform seed level action such as claim_by_seed, stake and un-stake seed. Cause in those actions, the contract has to iterator all farms under that seed. And if still some unclaimed reward in those farms, it would be going to an inner beneficiary account.  

It is a little like expiration but I choose to let DAO to decide when an ended farm will be expired.  Maybe I like the flexibility that a DAO can show.